code/game/machinery/computer/law.dm

```diff
@@ -27,9 +39,17 @@
 			current = null
 			return
 		M.install(current.laws, user)
+		if(alert("Do you wish to scramble the upload code?", "Scramble Code", "Yes", "No") == "No")
+			return
+		scramble_upload_code()
+		to_chat(usr, "<span class='notice'>You scramble the upload code</span>")
 	else
 		return ..()
 
+/proc/scramble_upload_code()
```

Comment:
```suggestion
/obj/machinery/computer/upload/proc/scramble_upload_code()
```

Reply:
This doesn't need to exist. Just call the proc without any wrappers.

Reply:
That proc is just unecessary yeah, no longer exists